pages/projects/submit.tsx

```diff
@@ -18,6 +19,8 @@ type FormInputs = {
   coverImage: string | null;
 };
 
+const fileTypes: string[] = ['JPG', 'PNG', 'GIF'];
```

Comment:
jpg file type works. but jpeg doesn't. can we add `jpeg` as well?

Reply:
yup on it

Reply:
done have a look